dataprofiler/tests/data/csv/num-negative-title-large.csv

```diff
@@ -0,0 +1,5 @@
+-12345
+11234
+34555
+12534
+02311
```

Comment:
why this file is different from the other file with smaller numbers?

Reply:
These numbers could have `4` selected as a number, there's also checks related to small fields (no / single character) vs more than one.